### PR TITLE
tidb-binlog: depreciate binlog as a data sync tool

### PR DESCRIPTION
**Pull request by benmaoer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-binlog/bidirectional-replication-between-tidb-clusters.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ aliases: ['/docs-cn/dev/tidb-binlog/bidirectional-replication-between-tidb-clust
 >
 > 目前双向同步属于实验特性，尚未经过完备的测试，不建议在生产环境中使用该功能。
 
+> **注意：**
+>
+> - TiDB Binlog 与 TiDB v5.0 版本开始引入的一些特性不兼容，无法一起使用，详情参照[注意事项](#注意事项)。
+> - 从 TiDB v7.5.0 版本开始，TiDB Binlog 组件的数据同步功能将不再获得技术支持，建议使用 [TiCDC](/ticdc/ticdc-overview.md) 作为数据同步的替代方案。
+> - 尽管在 TiDB v7.5.0 版本中仍为 TiDB Binlog 组件的实时备份和恢复提供技术支持，但该组件将在未来版本中完全废弃，建议尽早使用 [PITR](/br/br-pitr-guide.md) 作为数据恢复的替代方案。
```
Comment (BornChanger) [inline tidb-binlog/bidirectional-replication-between-tidb-clusters.md:16]:
```suggestion
> - 尽管在 TiDB v7.5.0 版本中仍为 TiDB Binlog 组件的实时备份和恢复提供技术支持，但该组件将在未来版本中完全废弃，建议尽早使用 [PiTR](/br/br-pitr-guide.md) 作为数据恢复的替代方案。
```

File: tidb-binlog/tidb-binlog-overview.md [hunk 2]
```diff
@@ -9,12 +9,14 @@ TiDB Binlog 是一个用于收集 TiDB 的 binlog，并提供准实时备份和
...
-- **实时备份和恢复**：备份 TiDB 集群数据，同时可以用于 TiDB 集群故障时恢复
+- **数据同步**：同步 TiDB 集群数据到其他数据库。
+- **实时备份和恢复**：备份 TiDB 集群数据，同时可以用于 TiDB 集群故障时恢复。
 
 > **注意：**
 >
-> TiDB Binlog 与 TiDB v5.0 版本开始引入的一些特性不兼容，无法一起使用，详情参照[注意事项](#注意事项)。建议使用 [TiCDC](/ticdc/ticdc-overview.md) 替代 TiDB Binlog。
+> - TiDB Binlog 与 TiDB v5.0 版本开始引入的一些特性不兼容，无法一起使用，详情参照[注意事项](#注意事项)。
+> - 从 TiDB v7.5.0 版本开始，TiDB Binlog 组件的数据同步功能将不再获得技术支持，建议使用 [TiCDC](/ticdc/ticdc-overview.md) 作为数据同步的替代方案。
+> - 尽管在 TiDB v7.5.0 版本中仍为 TiDB Binlog 组件的实时备份和恢复提供技术支持，但该组件将在未来版本中完全废弃，建议尽早使用 [PITR](/br/br-pitr-guide.md) 作为数据恢复的替代方案。
```
Comment (BornChanger) [inline tidb-binlog/tidb-binlog-overview.md:19]:
```suggestion
> - 尽管在 TiDB v7.5.0 版本中仍为 TiDB Binlog 组件的实时备份和恢复提供技术支持，但该组件将在未来版本中完全废弃，建议尽早使用 [PiTR](/br/br-pitr-guide.md) 作为数据恢复的替代方案。
```

File: tidb-binlog/bidirectional-replication-between-tidb-clusters.md (hunk 1 above)
Comment (Benjamin2037) [inline tidb-binlog/bidirectional-replication-between-tidb-clusters.md:16]: 需要给出 PiTR 的部分支持的规格 100T 么，这样其实可以加快用户的迁移，提升废弃速度。

**Review by lichunzhu — APPROVED** [review]
LGTM

File: tidb-binlog/bidirectional-replication-between-tidb-clusters.md (hunk 1 above)
Comment (Frank945946) [reply tidb-binlog/bidirectional-replication-between-tidb-clusters.md:16]: 未来版本有没有一个大致的时间，比如 7.5 之后的下一个 LTS 版本，这样用户心理有一个预期。

Comment (Oreoxmt) [inline tidb-binlog/bidirectional-replication-between-tidb-clusters.md:16]:
```suggestion
>
> - 目前双向同步属于实验特性，尚未经过完备的测试，不建议在生产环境中使用该功能。
> - TiDB Binlog 与 TiDB v5.0 开始引入的一些特性不兼容，无法一起使用，详情参照[注意事项](#注意事项)。
> - 从 TiDB v7.5.0 开始，TiDB Binlog 组件的数据同步功能不再提供技术支持，强烈建议使用 [TiCDC](/ticdc/ticdc-overview.md) 作为数据同步的替代方案。
> - 尽管 TiDB v7.5.0 仍支持 TiDB Binlog 组件的实时备份和恢复，但该组件在未来版本中将被完全废弃，推荐使用 [PITR](/br/br-pitr-guide.md) 作为数据恢复的替代方案。
```

File: tidb-binlog/tidb-binlog-overview.md [hunk 3]
```diff
@@ -9,12 +9,14 @@ TiDB Binlog 是一个用于收集 TiDB 的 binlog，并提供准实时备份和
 
 TiDB Binlog 支持以下功能场景：
 
-- **数据同步**：同步 TiDB 集群数据到其他数据库
-- **实时备份和恢复**：备份 TiDB 集群数据，同时可以用于 TiDB 集群故障时恢复
+- **数据同步**：同步 TiDB 集群数据到其他数据库。
+- **实时备份和恢复**：备份 TiDB 集群数据，同时可以用于 TiDB 集群故障时恢复。
```
Comment (Oreoxmt) [inline tidb-binlog/tidb-binlog-overview.md:13]:
```suggestion
- **数据同步**：同步 TiDB 集群数据到其他数据库。（从 v7.5.0 开始废弃）
- **实时备份和恢复**：备份 TiDB 集群数据，同时可以用于 TiDB 集群故障时恢复。
```
是否需要删除“数据同步”相关的描述？

File: tidb-binlog/bidirectional-replication-between-tidb-clusters.md (hunk 1 above)
Comment (benmaoer) [reply tidb-binlog/bidirectional-replication-between-tidb-clusters.md:16]: @BornChanger 所有对外的文档里都是 PITR，i 不小写，之前已经全部更新过一次

Comment (benmaoer) [reply tidb-binlog/bidirectional-replication-between-tidb-clusters.md:16]: 100 TB 咱们也无法说完全支持，建议在明确支持前，不要明确说支持 100 TB

File: tidb-binlog/tidb-binlog-overview.md (hunk 3 above)
Comment (benmaoer) [reply tidb-binlog/tidb-binlog-overview.md:13]: 不删除，在下面部分的注意事项中说明了不提供技术支持的版本，因为功能没有整体废弃，所以在数据同步这里说明废弃是不太合适。只是说不提供数据同步技术支持。

File: tidb-binlog/tidb-binlog-overview.md (hunk 2 above)
Comment (Oreoxmt) [inline tidb-binlog/tidb-binlog-overview.md:19]:
```suggestion
> - TiDB Binlog 与 TiDB v5.0 开始引入的一些特性不兼容，无法一起使用，详情参照[注意事项](#注意事项)。
> - 从 TiDB v7.5.0 开始，TiDB Binlog 组件的数据同步功能不再提供技术支持，强烈建议使用 [TiCDC](/ticdc/ticdc-overview.md) 作为数据同步的替代方案。
> - 尽管 TiDB v7.5.0 仍支持 TiDB Binlog 组件的实时备份和恢复，但该组件在未来版本中将被完全废弃，推荐使用 [PITR](/br/br-pitr-guide.md) 作为数据恢复的替代方案。
```

File: tidb-binlog/upgrade-tidb-binlog.md [hunk 4]
```diff
@@ -7,6 +7,12 @@ aliases: ['/docs-cn/dev/tidb-binlog/upgrade-tidb-binlog/','/docs-cn/dev/referenc
 
 如未特别指明，文中出现的 TiDB Binlog 均指最新的 [Cluster](/tidb-binlog/tidb-binlog-overview.md) 版本。
 
+> **注意：**
+>
+> - TiDB Binlog 与 TiDB v5.0 版本开始引入的一些特性不兼容，无法一起使用，详情参照[注意事项](#注意事项)。
+> - 从 TiDB v7.5.0 版本开始，TiDB Binlog 组件的数据同步功能将不再获得技术支持，建议使用 [TiCDC](/ticdc/ticdc-overview.md) 作为数据同步的替代方案。
+> - 尽管在 TiDB v7.5.0 版本中仍为 TiDB Binlog 组件的实时备份和恢复提供技术支持，但该组件将在未来版本中完全废弃，建议尽早使用 [PITR](/br/br-pitr-guide.md) 作为数据恢复的替代方案。
```
Comment (Oreoxmt) [inline tidb-binlog/upgrade-tidb-binlog.md:14]:
```suggestion
> - TiDB Binlog 与 TiDB v5.0 开始引入的一些特性不兼容，无法一起使用，详情参照[注意事项](#注意事项)。
> - 从 TiDB v7.5.0 开始，TiDB Binlog 组件的数据同步功能不再提供技术支持，强烈建议使用 [TiCDC](/ticdc/ticdc-overview.md) 作为数据同步的替代方案。
> - 尽管 TiDB v7.5.0 仍支持 TiDB Binlog 组件的实时备份和恢复，但该组件在未来版本中将被完全废弃，推荐使用 [PITR](/br/br-pitr-guide.md) 作为数据恢复的替代方案。
```

File: tidb-binlog/bidirectional-replication-between-tidb-clusters.md [hunk 5]
```diff
@@ -7,7 +7,10 @@ aliases: ['/docs-cn/dev/tidb-binlog/bidirectional-replication-between-tidb-clust
 
 > **警告：**
 >
-> 目前双向同步属于实验特性，尚未经过完备的测试，不建议在生产环境中使用该功能。
+> - 目前双向同步属于实验特性，尚未经过完备的测试，不建议在生产环境中使用该功能。
+> - TiDB Binlog 与 TiDB v5.0 开始引入的一些特性不兼容，无法一起使用，详情参照[注意事项](#注意事项)。
```
Comment (Oreoxmt) [inline tidb-binlog/bidirectional-replication-between-tidb-clusters.md:11]:
```suggestion
> - TiDB Binlog 与 TiDB v5.0 开始引入的一些特性不兼容，无法一起使用，详情参照[注意事项](/tidb-binlog/tidb-binlog-overview.md#注意事项)。
```

**Review by Oreoxmt — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**